.github/actions/argus-builder/build-prep/action.yml

```diff
@@ -120,4 +134,5 @@ runs:
         script: |
           const branchMatched = `${{ steps.branch_filter.outputs.match }}` === 'true';
           const filesMatched = `${{ steps.file_filter.outputs.run_on }}` === 'true';
-          core.setOutput('should_build', filesMatched && branchMatched);
+          const prHasStackLabel = `${{ steps.require_stack_label.outputs.has_label }}` === 'true';
+          core.setOutput('should_build', filesMatched && branchMatched && prHasStackLabel);
```

Comment:
note this also skips `Update Manifest`

is that okay or should I modify the manifest-update action to only skip the step that adds the values.yaml change?

Reply:
oh wait that step only writes the values.yml changes so it's safe to skip